.travis.yml

```diff
@@ -24,6 +24,28 @@ matrix:
       env: TOXENV=pep8py2
     - python: 3.6
       env: TOXENV=pep8py3
+    - python: 3.6
+      env: TOXENV=py36
+      apt:
+        packages:
+          - git
+          - gcc
+          - libssl-dev
+          - gnutls-bin
+      before_script:
+        - wget https://dist.opendnssec.org/source/softhsm-2.5.0.tar.gz \
+          && tar -xzf softhsm-2.5.0.tar.gz \
+          && cd softhsm-2.5.0 \
+          && ./configure --prefix=/usr \
```

Comment:
This will break in local user tests, needs to install in a dir relative to the base dir so that non-privileged builds work.
Also we shouldn't trash user's systems if they are running tests as root for whatever reason.